packages/edit-site/src/components/post-list/style.scss

```diff
@@ -64,6 +70,26 @@
 	overflow: hidden;
 }
 
+.dataviews-view-grid__primary-field.dataviews-view-grid__primary-field--clickable
```

Comment:
We want these styles to be applied to the title _only_ if the layout supports the item being "clickable". Either we assume these classes become public API (so they should be stable) or we need to figure out better ways to do this.